scripts/ohno.js

```diff
@@ -18,23 +18,16 @@ const ohno = (p) => {
     p.sendMessage('[scarlet]⚠[yellow]Sorry, the max number of ohno units has been reached.');
     return;
   }
-  const existing = []; // Keep track of spawned units to avoid changing wrong ones
-
-  Groups.unit.each((u) => {
-    // loop through all units and save their id
-    if (u.type === UnitTypes.atrax) {
-      existing.push(u.id);
-    }
-  });
-
-  UnitTypes.atrax.spawn(Team.sharded, p.unit().x, p.unit().y); // spawn unit at player's position
+  if(Vars.indexer.findEnemyTile(p.team(), p.unit().x, p.unit().y, 8, () -> true) != null){
```

Comment:
`->` is a syntax error in js. arrow functions are =>

Otherwise this is a really cool function. Didnt know about this

Reply:
Bruh
copypaste fail
thanks for the catch